sampletester/testplan.py

```diff
@@ -263,34 +267,30 @@ def accept(self, visitor: Visitor):
     return visitor.end_visit()
 
 
-def suite_configs_from(test_files):
-  """Returns the suite configs (key/value pairs) from all the `test_files`.
+SCHEMA_TYPE_VALUE = 'test'
```

Comment:
Unsure about Python convention, but does it make sense to pull `SCHEMA_TYPE_VALUE` out to a separate file dedicated for meta definitions?
For example,
```py
TEST_SCHEMA_TYPE_VALUE = 'test'
MANIFEST_SCHEMA_TYPE_VALUE = 'manifest'
SAMPLE_SCHEMA_TYPE = '...'
```

Reply:
I'm inclined to not do that, for modularity. The current way, each module can define its own `SCHEMA_TYPE_VALUE` without depending on a central file, and these values can be referenced via the module name (ie `foobar.SCHEMA_TYPE_VALUE` or imported in a very clear way like we do in inputs.py. An advantage to a central location is avoiding collisions, to be sure, but if/when we get complex enough to do that, my inclination would be to have a central registry that each module calls; the symbols would still be defined from each module.

Reply:
This is me thinking as a Java programmer and probably it's totally fine in the Python world. I'm OK with each module defining its own schema type value, but currently there's nothing enforcing the modules to define the schema type value this way (through `SCHEMA_TYPE_VALUE` module-wide global variable). In Java world, I would make test and manifest implementing the a common interface, which provides a `getSchemaTypeValue` method.

What happens if there is a typo, and foobar.SCHEMA_TYPE_VALUE can't be found? Would the error be straightforward enough?